DCOS/utils/wsmancmd.py

```diff
@@ -29,11 +29,11 @@
 
 
 def print_usage():
-    print ("%s [-U <url>] [-H <host>] [-P <port>] [-s] "
-           "[-a <basic|kerberos|certificate>] "
-           "[-u <username>] [-p <password>] "
-           "[-c <client_cert_pem> -k <client_cert_cert_key_pem>] "
-           "<cmd> [cmd_args]" % sys.argv[0])
+    print("%s [-U <url>] [-H <host>] [-P <port>] [-s] "
+          "[-a <basic|kerberos|certificate>] "
+          "[-u <username>] [-p <password>] "
+          "[-c <client_cert_pem> -k <client_cert_cert_key_pem>] "
+          "--file <powershell_script> --powershell <cmd> [cmd_args]" % sys.argv[0])
```

Comment:
I would say we leave the `--file` generic. If somebody wants to execute a remote script (without PowerShell) from file, he can do so.